CHANGES.md

```diff
@@ -22,6 +22,20 @@ OpenSSL 3.0
 -----------
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
+
+ * The EVP_PKEY_public_check() and EVP_PKEY_param_check() functions now work for
+   more key types include RSA, DSA, ED25519, X25519, ED448 and X448. Previously
+   (in 1.1.1) they would always return -2. For key types that do not have
+   parameters then EVP_PKEY_param_check() will still return 1.
+
+ * The output from numerous "printing" functions such X509_signature_print(),
+   X509_print_ex(), X509_CRL_print_ex(), and other similar functions has been
```

Comment:
has=have

Reply:
Hmm, no? You was say "The output has been", not "The output have been".